build.gradle

```diff
@@ -1,14 +1,29 @@
+
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4"
+  }
+}
+
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.bintray'
+
 allprojects {
   apply plugin: 'idea'
   apply plugin: 'eclipse'
   apply plugin: 'java'
   apply plugin: 'checkstyle'
 
-  version = "1.0.0"
+  version = "2.0.0"
```

Comment:
if we're usingthe semantic build versioning plugin than we dont want to hardcode a version

Reply:
removed hard coded version